# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/moz.build
# Commit: 59fe7c6fccbb
# Full Hash: 59fe7c6fccbbaf5d7c26e48c20a84f00a462519b
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Fix AudioConduit unittests. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D105771
# ==============================================================================

diff -r 49323e9c1054 -r 59fe7c6fccbb media/webrtc/signaling/gtest/moz.build
--- a/media/webrtc/signaling/gtest/moz.build	Tue Feb 16 18:03:06 2021 +0100
+++ b/media/webrtc/signaling/gtest/moz.build	Wed Feb 17 15:27:17 2021 +0100
@@ -39,7 +39,7 @@
     ]
 
     SOURCES += [
-#        "audioconduit_unittests.cpp",
+        "audioconduit_unittests.cpp",
         "jsep_session_unittest.cpp",
         "jsep_track_unittest.cpp",
         "mediapipeline_unittest.cpp",
